scripts/sync.sh

```diff
@@ -21,7 +21,23 @@ set -e
 cargo xtask update-apis
 
 book_example() {
-  sed '/ANCHOR/d' book/src/applet/prelude/$1.rs > examples/rust/$2/src/lib.rs
+  local src=book/src/applet/prelude/$1.rs
+  local dst=examples/rust/$2/src/lib.rs
+  if [ -z "$CI" -a $dst -nt $src ]; then
```

Comment:
Actually the problem occurs when cloning a repository (and probably also when changing branch). We don't control in which order git will update them. So we need to compare with a delta (dst is not newer than src by more than 2 seconds).